pub-matic-htb.js

```diff
@@ -251,6 +359,14 @@ function PubMaticHtb(configs) {
 
             var curBid;
 
+            if(!bids || bids.length === 0) {
```

Comment:
Can you please confirm that the response is always an array? Otherwise length may fail